test/using_witnet.js

```diff
@@ -97,6 +103,11 @@ contract("Using witnet", accounts => {
       assert(timestamp)
       assert.equal(pkh, accounts[0])
 
+      // Report block
+      blockRelay.postNewBlock(expectedBlockHash, expectedBlockHash, expectedBlockHash, {
```

Comment:
Is better to create new hashes for drMerkleRoot and TallyMerkleRoot instead of use the block hash for everyone, it is confusing